locker_service.py

```diff
@@ -12,10 +13,17 @@ def main():
             # scan id
             id, text = reader.read()
             print("id: " + str(id))
+
             # look for id in the db
             res = cursor.execute(
                 "select locker_id from packages where student_id = '" + str(id) + "'")
             locker_ids = res.fetchall()
+
+            # get all other information for logs
+            info = cursor.execute(
+                "select * from packages where student_id = '" + str(id) + "' and locker_id = '" + str(locker_ids[0][0]) + "'")
+            package_data = info.fetchall()
+
             if (locker_ids):
                 for locker_id in locker_ids:
```

Comment:
This loops goes through all packages that match the student_id input. So you'll need to change the variable names here to `for package in packages` to match the previous changes.

Reply:
by packages do you mean package_data?